src/dcp/dcp.c

```diff
@@ -8,9 +8,91 @@
 #include <errno.h>
 #include <stdbool.h>
 
+/* for daos */
+#ifdef DAOS_SUPPORT
+#include <uuid/uuid.h>
+#include <gurt/common.h>
+#include <gurt/hash.h>
+#include <daos.h>
+#include <daos_fs.h>
+#include <daos_uns.h>
+#endif
+
 #include "mpi.h"
 #include "libcircle.h"
-#include "mfu.h"
+#include "mfu.h" 
+
+#ifdef DAOS_SUPPORT
+static void daos_set_paths(int* rank, struct duns_attr_t dattr, struct duns_attr_t ddattr,
+			   char** argpaths, uuid_t* src_pool_uuid, uuid_t* src_cont_uuid,
+                           uuid_t* dst_pool_uuid, uuid_t* dst_cont_uuid, char* dfs_prefix,
+                           mfu_file_t* mfu_src_file, mfu_file_t* mfu_dst_file)
+{
+    /* find out if a dfs_prefix is being used,
+     * if so, then that means that the container
+     * is not being copied from the root of the
+     * UNS path  */
+    int rc = 0;
+    char* src_path = argpaths[0];
+    char* dst_path = argpaths[1];
```

Comment:
Can DAOS handle the user giving more than one source path, as in ```dcp src1 src2 dest/```?  If not, we should check for that and throw an error,

Reply:
@adammoody No it will only support one source.